src/Microsoft.DncEng.SecretManager/Readme.md

```diff
@@ -245,3 +290,27 @@ parameters:
   accountName: full account name including domain
   description: additional description for rotation of the password
 ```
+
+### Azure Active Directory Appljcation
+```yaml
+type: ad-application
+```
+
+### Azure DevOps Access Token
+```yaml
+type: azure-devops-access-token
+parameters:
+  organization: azure devops org
+  requiredScopes: string describing the required scopes
+  # only one of the following is required
+  domainAccountSecret: secret reference to a domain-account
+  gitHubBotAccountSecret: secret reference to a github-account
```

Comment:
Guessing this is specific to the indirect tokens we generate, but could it get more details why gitHubBotAccountSecret is a parameter in AzDO access token?

Reply:
Its possible to log in to azdo with a github account.

Reply:
ah, I did not know.